src/components/map/LocationOverview.tsx

```diff
@@ -36,14 +38,21 @@ const mapToMapMarkers = (data: any) => {
 };
 
 export const LocationOverview = ({ navigation, category }: Props) => {
+  const { isConnected, isMainserverUp } = useContext(NetworkContext);
   const [selectedPointOfInterest, setSelectedPointOfInterest] = useState<string>();
 
+  const fetchPolicy = graphqlFetchPolicy({ isConnected, isMainserverUp, refreshTime: undefined});
```

Comment:
this could be `const fetchPolicy = graphqlFetchPolicy({ isConnected, isMainserverUp })` like in other places already. The `refreshTime: undefined` is not necessary, if there is no refresh time value to pass.

Reply:
it is necessary in this case because typescript does not know that this is an optional key of the argument.